<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FFmpegBuildTool</class>
 <widget class="QWidget" name="FFmpegBuildTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>FFmepg Select</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="2">
         <widget class="QLineEdit" name="remote_ffmpeg_url_le">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Local FFmpeg:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Remote FFmpeg:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="remote_ffmpeg_title_combx">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QComboBox" name="local_ffmpeg_combx">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="select_local_path_btn">
          <property name="text">
           <string>Select</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Download</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Select Options</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,1,1" columnstretch="1,0">
        <item row="0" column="0">
         <widget class="QCheckBox" name="show_cmd_cbx">
          <property name="text">
           <string>Show Cmd</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="build_btn">
          <property name="text">
           <string>Build</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="InfoWidgets" name="select_option_tb" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="ZTextEditor" name="cmd_ple">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Complie Options</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="TabConfigWg" name="ffmpeg_complie_options_tb" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Complie Outputs</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="ZTextEditor" name="complie_output_ple"/>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InfoWidgets</class>
   <extends>QWidget</extends>
   <header>widgets/infowidgets.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabConfigWg</class>
   <extends>QWidget</extends>
   <header>widgets/tabconfigwg.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ZTextEditor</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">common/ztexteditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
